src/streamsync/app_runner.py

```diff
@@ -539,7 +539,7 @@ def __init__(self, app_path: str, mode: str):
         self.is_app_process_server_ready = multiprocessing.Event()
         self.is_app_process_server_failed = multiprocessing.Event()
         self.app_process_listener: Optional[AppProcessListener] = None
-        self.observer: Optional[watchdog.observers.Observer] = None
+        self.observer: Optional[PollingObserver] = None
```

Comment:
Was this intentional?

Reply:
It is, otherwise `mypy` was complaining in `build.sh`

Reply:
Ah nice, must be related to the dependencies I upgraded, thanks for fixing.